Prog/Sec8MF/Sec8MF_2017_08.sas

```diff
@@ -0,0 +1,54 @@
+/**************************************************************************
+ Program:  Sec8MF_yyyy_mm.sas
```

Comment:
Please update the program name in the header. Also, similar to the MFIS program, the program name should align with the **date of the update file** that we are applying. So, if the update file is 9/1/17, then the program name should be **Sec8mf_2017_09.sas**. Please rename the file.